src/mapgen_functions.cpp

```diff
@@ -1245,58 +1245,60 @@ void mapgen_subway( map *m, oter_id terrain_type, mapgendata dat, const time_poi
     switch( num_dirs ) {
         case 4: // 4-way intersection
             mapf::formatted_set_simple( m, 0, 0, "\
-.DD^^DD^.######.^DD^^DD.\n\
-DD^^DD^..######..^DD^^DD\n\
-D^^DD^....####....^DD^^D\n\
-^^DD^..............^DD^^\n\
-^DD^................^DD^\n\
-DD^..................^DD\n\
-D^....................^D\n\
-........................\n\
-........................\n\
-##........####........##\n\
-###......######......###\n\
-###......######......###\n\
-###......######......###\n\
-###......######......###\n\
-##........####........##\n\
-........................\n\
-........................\n\
-D^....................^D\n\
-DD^..................^DD\n\
-^DD^................^DD^\n\
-^^DD^..............^DD^^\n\
-D^^DD^....####....^DD^^D\n\
-DD^^DD^..######..^DD^^DD\n\
-.DD^^DD^.######.^DD^^DD.",
-                                        mapf::ter_bind( ". # ^ D",
+..^/D^^/D^....^D/^^D/^..\n\
```

Comment:
```suggestion
                                           "..^/D^^/D^....^D/^^D/^..\n"
```
for this an every line afterwards - it looks nicer.